Cargo.toml

```diff
@@ -9,3 +9,6 @@ members = [
 debug = true
 panic = 'unwind'
 incremental = false
+ 
+[profile.dev]
+split-debuginfo = "unpacked" 
```

Comment:
Does this comes with some performance benefits for debug compilation? Is there a practical advantage of `unpacked` VS other options?

Reply:
Yes, it it said to enable faster debug builds on MacOS (it used to use a slow tool for incremental builds by default), but by [digging deeper](https://github.com/rust-lang/cargo/pull/9112) I found out that it should not be necessary with newer rustc versions anymore. 👍